detox/runners/jest/DetoxAdapterCircus.js

```diff
@@ -35,7 +35,7 @@ class DetoxAdapterCircus extends CircusTestEventListenerBase {
 
   _onTestStart(event) {
     const { test } = event;
-    if (test.mode === 'skip' || test.mode === 'todo') {
+    if (test.mode === 'skip' || test.mode === 'todo' || test.errors.length > 0) {
```

Comment:
Here I am making sure that artifacts manager does not receive messages about `test start` when it is skipped because of the outer timeout.

Reply:
So Jest runs the tests nonetheless even with Circus but at also delivers errors as an indication?